### PR TITLE
cleaning up

### PR DESCRIPTION
**Pull request by Simonolsson98**

### CODE REVIEW

File: KA-Flocking/Assets/Scripts/FlockBehaviour.cs [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-    public abstract Vector3 CalculateMove (FlockAgent agent, List<Transform> context, Flock flock);
-}
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public abstract class FlockBehaviour : ScriptableObject
+{
+    public abstract Vector3 CalculateMove (FlockAgent agent, List<Transform> context, Flock flock);
+}
```
Comment (Hoidi) [inline KA-Flocking/Assets/Scripts/FlockBehaviour.cs]: What happened here? Looks like nothing 🤔

Comment (Hoidi) [reply KA-Flocking/Assets/Scripts/FlockBehaviour.cs]: Same with all the other `.cs` files. I think you just stole all of @jerge's code ☹️

Comment (Simonolsson98) [reply KA-Flocking/Assets/Scripts/FlockBehaviour.cs]: my code now yoink

Comment (Simonolsson98) [reply KA-Flocking/Assets/Scripts/FlockBehaviour.cs]: vette fan varför de blir så lul i didnt touch this

**Pull request merged**